gremlin-python/src/main/jython/tests/structure/io/test_graphson.py

```diff
@@ -121,6 +125,19 @@ class X(object):
         serdes.objectify.assert_called_once_with(value, reader)
         assert o is serdes.objectify()
 
+    def test_datetime(self):
+        dt = self.graphson_reader.readObject(json.dumps({"@type": "g:Date", "@value": 1481750076295}))
+        assert isinstance(dt, datetime.datetime)
```

Comment:
Wouldn't it make sense to also assert that the value is correct here? Then the test would validate the GraphSON reader completely and it stays consistent with the other tests in this class.

Reply:
Yeah I guess. TBH, I don't find any of these tests too meaningful, which is why I added the functional IO tests at the end that actually send and receive messages from the server and verify that they are equal. I can add the asserts though if you want.

Reply:
Done